packages/styles/components/_c.tooltip.scss

```diff
@@ -110,6 +134,12 @@
         right: 0;
         transform: translate(99%, -50%) rotate(180deg);
       }
+
+      &::before {
+        top: 50%;
+        right: -1px;
+        transform: translate(99%, -50%) rotate(180deg);
+      }
```

Comment:
The values of the `::before` should be the same as the `::after`.
_(same remark as above)_